src/main/java/Duke/tasks/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package Duke.tasks;
+
+public class Deadline extends Task {
+    private String time;
+
+    public Deadline(String answer){
+        String[] sentences=answer.split("/by ");
+        time=sentences[1];
+        String[] name=sentences[0].split("deadline ");
+        super.modifyName(name[1]);
+        System.out.println("\n\tGot it. I've added this task:");
+        System.out.println("\t  "+this);
+    }
+
+    public String toString() {
+        if (super.getDone()) return String.format("[D][✓] "+super.getName()+"(by: "+time+")");
```

Comment:
Same issue as previously mentioned. Your `if-else` structure should follow that of the Java coding standard.